Codecademy/gamut
# fix(InfoTip): fix DOM ordering + tip not being reread

### Overview


Fixes DOM ordering and Infotip not being re-read on retrigger (known issue with aria-live on Chrome)


i've found that properly fixing the floating alignment would require an entire overhaul of the Popover component -- i've caught a larger epic for that [here](https://skillsoftdev.atlassian.net/jira/software/projects/GM/boards/784?assignee=712020:d3cb9076-03cb-43eb-905b-058aa25d1578&selectedIssue=GM-788).

 instead, the user can tab through the **_screenreader_** text for the floating component which does respect DOM order - giving the user the correct information in the correct order even if its not the same HTML element. this is a quick fix, i'm planning for a long-term fix with the adjustments to the Popover component but thats a much larger effort.

### PR Checklist

- [x] Related to JIRA ticket: [gm-788, [ENT-73](https://skillsoftdev.atlassian.net/browse/ENT-73?focusedCommentId=731387)]

#### Testing Instructions



1. Head to InfoTip story
2. Turn on VO 
3. Trigger an InfoTip at least 3x times - you should hear it read each re-trigger
4. Navigate to the alignment section
5. Open a top-aligned infotip
6. You should be able to hear the tip announced, and navigate through the text in correct DOM order using CTRL + OPT
7. Open a bottom-aligned infotip
8. You should be able to hear the tip announced, and navigate through the text in correct DOM order using CTRL + OPT
9. Navigate to the floating section
10. You should be able to hear the tip announced, and navigate through the text in correct DOM order using CTRL + OPT. The tip **will not be highlighted**, though the information should be communicated in the correct order.
11. Navigate to ToolTip page
12. You should only hear tips communicated as the Buttons and not repeated when navigating with CTRL + OPT
13. Profit

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/38063/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/6972')  | [Portal Env](http://www.google.fr/ 'Named link title')   |




[ENT-73]: https://skillsoftdev.atlassian.net/browse/ENT-73?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
